db/seeds.rb

```diff
@@ -1,12 +1,14 @@
 if Rails.env.development?
```

Comment:
You mean something like `I_UNDERSTAND_THIS_IS_GOING_TO_ADD_SEED_DATA_TO_PRODUCTION_DB=true rake db:seed`?